data/CMakeLists.txt

```diff
@@ -4,4 +4,5 @@
 
 add_subdirectory(yarpmanager)
 add_subdirectory(yarpdataplayer)
+add_subdirectory(harness_dev)
```

Comment:
Hi @drdanz, 
we added this directory containing the ini files for running `TestFrameGrabberTest`, that needs them to use the new `ServerGrabber`.
We created this context for `harness_dev`, where should it be ?